libraries/networking/src/AssetClient.cpp

```diff
@@ -57,34 +55,6 @@ AssetClient::AssetClient() {
             this, &AssetClient::handleNodeClientConnectionReset);
 }
 
-void AssetClient::initCaching() {
```

Comment:
more info below, but for some reason, it seems like we were only setting up caching for this AssetClient, not for all other requests?  this code moved to NetworkAccessManager::getInstance()